### DISCUSSION TITLE
How to declare a non reactive variable with object syntax

### DISCUSSION THREAD

**Discussion by Rolanddoda** [vuejs/pinia#1220] [Help and Questions] [answered]
With function syntax, you can do

```js
const useMyStore = defineStore('myStore', () => {
 const nonReactive = getValueFromSomeFunction()

 return {
  nonReactive
 }
})
```

**Is there any way to do this with object syntax?**

---------------------

- *I am not trying to achieve something, I am just exploring Object and Function syntax.*
- *nonReactive can be any type of value primitive or not*

**Comment by posva** [comment, marked as answer]
Use [markRaw()](https://vuejs.org/api/reactivity-advanced.html#markraw)

```ts
state: () => ({ nonReactive: markRaw(...) })
```
